x/upgrade/abci_test.go

```diff
@@ -236,17 +236,17 @@ func TestPlanStringer(t *testing.T) {
 	require.Equal(t, `Upgrade Plan
   Name: test
   Time: 2020-01-01T00:00:00Z
-  Info: 
+  Info: .
```

Comment:
I'm curious if you know which change in the code resulted in these output changes?

Reply:
it seems like some tests are failing because of this.

Reply:
Yes, editor removed the trailing whitespace

Reply:
I can put it back. Adding a dot was easier.